Makefile

```diff
@@ -52,7 +52,7 @@ shellcheck: ## run shellcheck validation
 .PHONY: fmt
 fmt: ## run gofumpt (if present) or gofmt
 	@if command -v gofumpt > /dev/null; then \
-		gofumpt -w -d -lang=1.21 . ; \
+		gofumpt -w -d -lang=1.23 . ; \
```

Comment:
nit: We could probably have a variable on the top of the file so it's more discoverable

Reply:
Hm. Yeah, I was considering a bit how we could improve this, and went a bit back-and-forth;

- It's not _really_ critical (switching did not result in a diff, so looks like there were no version-specific changes)
- It should no longer be needed when we have a `go.mod`, so .. 🤞 should go away in the near future (so didn't want to add the extra code-churn)
- _but_ I think we should look if we can containerise this (add a `docker bake` target); that way we would remove the requirement to have `gofumpt` installed, and make sure that we run it in a consistent environment (and expected version of go)
- ☝️ similar to how we do vendoring
- Was actually asking @crazy-max Yesterday (i.e., should we have some templates for such things?); related to https://github.com/docker/compose/pull/12392#issuecomment-2548193305